src/routes/Details.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCountry } from '../redux/countriesSlice/countriesSlice';
+import './Details.scss';
+import Navbar from '../components/Navbar/Navbar';
+
+const Details = () => {
+  const { countrySearch } = useSelector((store) => store.country);
+  const dispatch = useDispatch();
+  const { code } = useParams();
+
+  useEffect(() => {
+    if (code) {
+      dispatch(getCountry(code.toLowerCase()));
+    }
+  }, [dispatch, code]);
+  return (
+    <div>
+      <Navbar title="Details" />
```

Comment:
- [ ] It would be better in your Details Navbar to change the home icon to a back icon so users know they are navigating back to home page, please update it so it creates a better user experience 👍🏿